include/Core/DB/TableManager.php

```diff
@@ -482,4 +491,67 @@ private function get_create_table_query() {
 
 		return $sql;
 	}
+
+	/**
+	 * Callback for the Array filter.
+	 *
+	 * @since 2.3.0
+	 *
+	 * @param   string $column A column from the array Columns.
+	 *
+	 * @return bool
+	 */
+	private function validate_columns( $column ) {
+		return in_array( $column, array( 'to' ), true );
+	}
+
+	/**
+	 * @since 2.3.0
+	 *
+	 * @param $columns
+	 *
+	 * @uses \EmailLog\Core\DB\TableManager::validate_columns()
+	 *
+	 * @return array|null|object|void
+	 */
+	public function fetch_log_items_for_edd_customer_tab_display( $columns ) {
```

Comment:
If the logic of this function is generic, then let's rename this function to a more generic name. Or if the logic is going to be EDD specific, then let's move this function to the Advanced search add-on itself.

I would prefer option 1 though.